### ISSUE TITLE
Eclipse hangs when importing eclipse project zip

### ISSUE THREAD

**Issue by billreed63** [maqetta/maqetta#1489]
Start maqetta in mutli user mode. (repeat tests in single user mode)
Create a new project by going to New-->Project in the menu bar.
Type a name for the project, and click the "Enable Eclipse Support"
Create a new folder under WebContent by going to the New-->Folder in the title bar. Call the folder aaa
Create a new HTML file in the WebContent/aaa folder by going to New-->HTML File in the menu bar
Add some content to the new HTML file. Save and close
Download the entire workspace as a zip by clicking the "Download Workspace" icon on the File Explore shortcut bar.
Default settings are fine, click OK and save the file somewhere on the file system
Start Eclipse. Then go to your workspace and right click and select Import-->Import
In "General"" select "Existing Project into Workspace", Click the radio button next to "From Archive" then browse to find the zip file you just downloaded.

Eclipse hangs with import dialog just spinning, I this point I have to kill eclipse

**Comment by childsb** [comment]
i can't reproduce this. will discuss on monday.

**Comment by JonFerraiolo** [comment]
I just tried this running the development version of Maqetta (i.e., launched from Eclipse) in single-user mode where I created a ZIP file in the Mac/Chrome downloads folders, and then I successfully imported from archive in Mac/Eclipse (Version: Helios Service Release 2 Build id: 20110218-0911). I'll try some other configurations...

There was one problem - there is a mysterious "unknown" folder in the root level of the project after importing into Eclipse. This is what I see in the Eclipse navigator:

```
myeproj
  JavaScript Resources
  undefined
  WebContent
    lib
    maqetta
    etc
```

**Comment by JonFerraiolo** [comment]
I was able to successfully import from the daily build running on lab9:50010 using Win/FF and importing a Maqetta/Eclipse project from a ZIP using Win/Eclipse (Version: Helios Service Release 2 Build id: 20110218-0911)

**Comment by childsb** [comment]
I resolved the "undefined" folder with commit e57f533dce686620d639f23a9f6d590381a0caa5

i'm going to close this issue for now.  please reopen if you encounter/can reproduce.
